types/params.go

```diff
@@ -77,7 +80,8 @@ func DefaultBlockParams() BlockParams {
 // DefaultEvidenceParams Params returns a default EvidenceParams.
 func DefaultEvidenceParams() EvidenceParams {
 	return EvidenceParams{
-		MaxAge: 100000, // 27.8 hrs at 1block/s
+		MaxAgeHeight:   100000, // 27.8 hrs at 1block/s
+		MaxAgeDuration: (48 * time.Hour),
```

Comment:
no need in curly braces

```suggestion
		MaxAgeDuration: 48 * time.Hour,
```